ivy_tests/test_ivy/test_frontends/test_torch/test_loss_functions.py

```diff
@@ -563,6 +563,79 @@ def test_torch_nll_loss(
     )
 
 
+# gaussian_nll_loss
+@handle_frontend_test(
+    fn_tree="torch.nn.functional.gaussian_nll_loss",
+    dtype_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_value=0.01,
+        max_value=1.0,
+        allow_inf=False,
+        min_num_dims=1,
+        max_num_dims=1,
+        min_dim_size=1,
+        max_dim_size=1,
+    ),
+    dtype_and_target=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer"),
+        min_value=0.0,
+        max_value=1.0,
+        allow_inf=False,
+        min_num_dims=1,
+        max_num_dims=1,
+        min_dim_size=1,
+        max_dim_size=1,
+    ),
+    dtype_and_var=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        allow_inf=False,
+        min_value=0.0,
+        min_num_dims=1,
+        max_num_dims=1,
+        min_dim_size=1,
+        max_dim_size=1,
+    ),
+    dtype_and_eps=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
Since `eps` is just a number you could use `st.floats` to generate it.

Reply:
Also, I'd say that a max value of 1 would make sense for `eps` as it's just there for numeric stability.

Reply:
Apparently `.clone()` is not implemented in Jax backend I guess and similarly .clamp_ is not implemented in torch backend I guess. These tow methods are raising errors from backend. I figured a roundabout way of doing same thing by suing `torch_frontend` functions and array object methods. Kindly review it and see If I have done it rightly.